tsconfig-common.json

```diff
@@ -8,7 +8,7 @@
         /* Modules */
         // Required for dynamic imports even though we aren't using
         // tsc to output any modules.
-        "module": "ESNext",
+        "module": "CommonJS",
```

Comment:
Is this comment still true?

Reply:
I can revert this change.  I was experimenting with things.  I don't believe that the settings in this tsconfig affect the babel settings.